packages/remediations/src/NewRemediationWizard.js

```diff
@@ -137,33 +140,54 @@ class RemediationWizard extends Component {
+        })
+    );
+
     render () {
 
         return (
             this.state.open ?
-                <RemediationWizardContext.Provider
-                    value={{ ...this.state.wizardContextValue,
-                        setWizardError: this.setWizardError,
-                        setWizardSuccess: this.setWizardSuccess,
-                        setHideForm: this.setHideForm }}>
-                    <FormRenderer
-                        schema={this.state.schema}
-                        container={this.container}
-                        subscription={{ values: true }}
-                        FormTemplate={this.getFormTemplate}
-                        initialValues={{
-                            multiple: !!this.state.resolutions?.find(r => r.resolutions.length > 1),
-                            'manual-resolution': true,
-                            'selected-resolutions': {}
-                        }}
-                        componentMapper={{
-                            [componentTypes.WIZARD]: Wizard,
-                            ...this.state.mapperExtension
-                        }}
-                        onSubmit={() => undefined}
-                        onCancel={this.closeWizard}
-                    />
-                </RemediationWizardContext.Provider>
+                <FormRenderer
+                    schema={this.state.schema}
+                    container={this.container}
+                    subscription={{ values: true }}
+                    FormTemplate={this.getFormTemplate}
+                    initialValues={{
+                        multiple: !!this.state.resolutions?.find(r => r.resolutions.length > 1),
+                        'manual-resolution': true,
+                        'selected-resolutions': {}
+                    }}
+                    componentMapper={{
+                        [componentTypes.WIZARD]: Wizard,
+                        ...this.state.mapperExtension
+                    }}
+                    onSubmit={(_, formOptions) => {
+                        const values = formOptions.getState().values;
+                        const issues = this.state.data.issues.map(({ id }) => ({
+                            id,
+                            resolution: values['manual-resolution'] ? this.getResolution(id)?.[0]?.id : undefined,
+                            systems: this.state.data.systems
+                        }));
+                        const add = { issues, systems: this.state.data.systems };
+                        if (values['existing-playbook-selected']) {
+                            const { id, name } = values['existing-playbook'];
+                            // eslint-disable-next-line camelcase
+                            api.patchRemediation(id, { add, auto_reboot: values['auto-reboot'] }, this.state.basePath)
+                            .then(() => this.resolver(this.state.deferred)(id, name, false));
```

Comment:
Resolver doesn't take any parameters, so you shouldn't pass it. Also is `this.state.deferred` actually necessary? If not, we might get rid of it.